meteor-app/imports/ui/Report_Builder/verifyReport.ts

```diff
@@ -0,0 +1,59 @@
+import { ReportStructure } from '../../api/types/reports'
+
+
+// {_id : Match.Maybe(String), account_id : Match.Maybe(String), name : String, 
```

Comment:
you can remove this commented out code, and if you need to reference the check statement on the server, 

instead leave a comment that points to that file, and method